docs/preview/features/logging.md

```diff
@@ -138,6 +139,33 @@ So the resulting log message becomes:
 
 `HTTP Request POST http://localhost:5000/weatherforecast completed with 200 in 00:00:00.0191554 at 03/23/2020 10:12:55 +00:00 - (Context: [X-Api-Key,])`
 
+## Tracking application version
+
+The `Arcus.WebApi.Logging` library allows you to add application version tracking to your <span>ASP.NET</span> application which will include your application version to a configurable response header.
+
+This functionality uses the `IAppVersion`, available in the [Arcus.Observability](https://observability.arcus-azure.net/features/telemetry-enrichment#version-enricher) library, for retrieving the current application version.
+Such an instance **needs** to be registered in order for the version tracking to work correctly.
+
+<div markdown="span" class="alert alert-danger" role="alert"><i class="fa fa-exclamation-circle"></i> <b>WARNING:</b> only use the version tracking for non-public endpoints otherwise can the leakage of the application version be used in unintended malicious purposes.</div>
```

Comment:
```suggestion
<div markdown="span" class="alert alert-danger" role="alert"><i class="fa fa-exclamation-circle"></i> <b>WARNING:</b> Only use the version tracking for non-public endpoints otherwise the version information is leaked and it can be used for unintended malicious purposes.</div>
```

Reply:
Or let's use native Markdown:
```suggestion
> ⚠ **Warning:** only use the version tracking for non-public endpoints otherwise can the leakage of the application version be used in unintended malicious purposes.
```